.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,16 @@
-#### Proposed Changes
+<!--
+Link a related issue to this PR. If the PR does not immediately resolve the issue,
+for example, it requires a separate deployment to production, avoid
+using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
+the linked issue.
+-->
+
+See #
```

Comment:
Nitpick: I feel that `Related to #` was more accurate, but that's just me 😅

```suggestion
Related to #
```

Reply:
> Nitpick: I feel that `Related to #` is more accurate, but that's just me

Sounds good, not a strong opinion on my part.